src/VisualStudio/Core/Def/Implementation/SymbolSearch/SymbolSearchResult.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
`#nullable enable` the file please, especially with the various bits here that have to deal with nulls.